### PR TITLE
tests for dot package support and add -p to build command

### PR DESCRIPTION
**Pull request by evansde77**
allow for separate package name, setting python interpreter

Continue to improve the container init command, include some additional options for controling package names and python interpreters

### CODE REVIEW

File: src/cirrus/package.py [hunk 1]
```diff
@@ -132,6 +132,11 @@ def build_parser(argslist):
         dest='desc',
         default='PACKAGE DESCRIPTION HERE'
     )
+    init_command.add_argument(
+        '--pypi-package-name',
+        help='Name for package on upload to pypi, use if different to package option',
```
Comment (PerilousApricot) [inline src/cirrus/package.py]: grammar? Should be 'different from'?

File: src/cirrus/package.py [hunk 2]
```diff
@@ -161,6 +166,11 @@ def build_parser(argslist):
         dest='test_requirements'
     )
     init_command.add_argument(
+        '--python',
+        help='optionally specifiy the name of python binary to use in this package, eg python2, python3',
```
Comment (PerilousApricot) [inline src/cirrus/package.py]: `s/specify/specifiy/`?

File: src/cirrus/package.py [hunk 3]
```diff
@@ -415,9 +425,12 @@ def write_cirrus_conf(opts, version_file):
     cirrus_conf = os.path.join(opts.repo, 'cirrus.conf')
     LOGGER.info("setting up cirrus.conf: {}".format(cirrus_conf))
     backup_file(cirrus_conf)
+    pname = opts.package
+    if opts.pypi_package_name:
+        pname =opts.pypi_package_name
```
Comment (PerilousApricot) [inline src/cirrus/package.py]: spacerino

File: src/cirrus/package.py (hunk 1 above)
Comment (evansde77) [inline src/cirrus/package.py]: will fix

**Comment by PerilousApricot** [comment]
Unsure of how this GH review feature works..

**Comment by evansde77** [comment]
You keep complaining, i keep fixing, eventually you type LGTM or +1 and I merge it

**Pull request merged**